document/PolicyComplianceProfile.mdk

```diff
@@ -1 +1,65 @@
-# Policy Compliance Profile { #policy-compliance-profile }
\ No newline at end of file
+# Policy Compliance Profile { #policy-compliance-profile }
```

Comment:
Does the Policy Compliance Profile also need to specify which file types were analyzed. e.g. I might have enabled 'do not use eval()', but configured the tool to not run on HTML files.